docs/reference/mapping/params/meta.asciidoc

```diff
@@ -29,3 +29,24 @@ than or equal to 50.
 
 NOTE: Field metadata is updatable by submitting a mapping update. The metadata
 of the update will override the metadata of the existing field.
+
+Elastic products standardize on the following metadata, which you may use too
+in order to get a better out-of-the-box experience with your data.
+
+unit::
+
+  The unit of the field. Recognizes `"byte"` as well as
+  <<time-units,time units>>. By default, fields are assumed to not have a unit.
+  This information should only be specified on numeric fields.
+
+metric_type::
+
+  The type of metric. Recognizes `"gauge"` and `"counter"`. By default, fields
+  are assumed to not be a metric. This information should only be specified on
+  numeric fields.
```

Comment:
I'd like to explain what gauges and counters are in here. Something like:

The type of metric for time series data. Options are none, `"gauge"` and `"counter"`.

A gauge is a number representing a point in time that can go up and down.

A counter is a number representing a cumulative amount and is always increasing. Counters can reset to 0 occasionally.